src/metaImage.cxx

```diff
@@ -449,10 +449,10 @@ void MetaImage::Clear(void)
 
   m_HeaderSize = 0;
 
-  memset(m_SequenceID, 0, 4*sizeof(float));
```

Comment:
also is the removal of `4` intentional ?

Reply:
seriously ?

Reply:
Makes sense.